src/main/java/by/mifort/automation/hr/dev/entity/CandidateAttributes.java

```diff
@@ -0,0 +1,94 @@
+package by.mifort.automation.hr.dev.entity;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+import javax.persistence.*;
+
+/**
+ * People attributes that connected with candidate
+ * @see AttributeTypes connected attributes with types
+ * @author yauheni_vozny
+ * @version 1.0
+ */
+
+@Entity
+@Table(name = "peopleattributes")
+public class CandidateAttributes {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    @Column(name = "value")
+    private String name;
+
+    @Column(name = "valuesource")
+    private Integer valueSource;
+
+    @ManyToOne
+    @JoinColumn(name = "user_id", nullable = false)
```

Comment:
we should use camelcase

Reply:
have some problems with this case in postgreSQL, i wrote userId, but postgreSQL ignored it, and wanted smth else, i talked with Andrey, and he said to make this in database and here

Reply:
ok, leave todo here